src/components/Donation/constants/chapters.ts

```diff
@@ -1,5 +1,5 @@
 export default [
-  { label: 'I am not part of a chapter', value: 'none' },
+  { label: 'I am not part of a organizing', value: 'none' },
```

Comment:
mmm I'm not sure about this, sounds weird to me

Reply:
@duranmla Actually, I think we need to remove the chapters dropdown completely, from both the donation form and the membership form. Also, these changes need to be ported to this repo https://github.com/debtcollective/packages/tree/development/packages/union-component where we have the components. There's a ticket to refactor the homepage to use this package instead of having the components inside this project

Reply:
This code no longer exists as I have migrated to the package after 06ccf87 as you mentioned before. Thanks!